src/main/java/Main.java

```diff
@@ -0,0 +1,21 @@
+import java.util.ArrayList;
+import java.util.NoSuchElementException;
+
+public class Main {
+
+    public static void main(String[] args) {
```

Comment:
It looks like, you made the `Main` class at first to test your implementation and then wrote the tests. It is easier to write tests and implementation simultaneously, you will see after a while=)